public-pool/docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: public-pool_proxy_1
+      APP_PORT: 80
+      PROXY_AUTH_WHITELIST: "/api/*"
+
+  web:
+    image: smolgrrr/public-pool-ui:0.1.0@sha256:e219808b92a9f52fc5938860a731d317cf6db1d059af3b982b02b48cbe06dc3b
+    restart: unless-stopped
+    stop_grace_period: 30s
+    environment:
+      - DOMAIN=$DEVICE_DOMAIN_NAME
+
+  server:
+    image: sethforprivacy/public-pool:4bb00f2@sha256:0fc82e7123be68650c9e31f57a5a206e622acbcaa170e31219a488818ad6da6b
+    restart: unless-stopped
+    stop_grace_period: 30s
+    ports:
+      - "0.0.0.0:2018:2018/tcp"
+    volumes:
+      - "${APP_DATA_DIR}/data/db/public-pool-db:/public-pool/DB"
```

Comment:
Need to add the db directory to this repo (under the `data` directory that you have the proxy dir in) and include a .gitkeep within it so that it gets committed to this github repo even when empty.

If you want you can simplify the bind mount to:
`- "${APP_DATA_DIR}/data/db:/public-pool/DB"`

Reply:
oops thanks missed that